src/app.js

```diff
@@ -12,6 +12,11 @@ const landenKaart = [ 'Heard Island and McDonald Islands', 'South Georgia and th
 ]
 
 
```

Comment:
Met ul { 
column-count: 3;
}
Is dit de nieuwe code:
**CODE HIERONDER**
import axios from 'axios';

async function getAllCountries() {

    try {
        const result = await axios.get('https://restcountries.com/v2/all');

        const allObjects = result.data;

        const allObjectsSorted = allObjects.sort(function(a,b) {
            return a.population - b.population
        })


        const allCountriesDiv = document.getElementById('allCountries');
        const makeCountryList = () => {
            const list1 = document.createElement('ul');
            list1.setAttribute('id', 'list1');
            allCountriesDiv.appendChild(list1);

            const listNameAndFlag = [];
            const listPopulation = [];
            const images = [];
            const names = [];
            const population = [];
            const region = [];
            const subregion = [];


            function colorCountriesPerContinent(region, i, subregion) {
                if (region[i] === 'Africa') {
                    listNameAndFlag[i].setAttribute('class', region[i]);
                } else if (region[i] === 'Asia') {
                    listNameAndFlag[i].setAttribute('class', region[i]);
                } else if (region[i] === 'Europe') {
                    listNameAndFlag[i].setAttribute('class', region[i]);
                } else if (region[i].includes('Antarctic') || region[i] === 'Polar') {
                    listNameAndFlag[i].setAttribute('class', 'Antarctic');
                } else if (region[i] === 'Oceania') {
                    listNameAndFlag[i].setAttribute('class', region[i]);
                } else if (region[i] === 'Americas') {
                    if (subregion[i].includes('South')) {
                        listNameAndFlag[i].setAttribute('class', 'SouthAmerica');
                    } else if (subregion[i].includes('North') || subregion[i] === 'Caribbean') {
                        listNameAndFlag[i].setAttribute('class', 'NorthAmerica');
                    }
                    else {
                        listNameAndFlag[i].setAttribute('class', 'NorthAmerica')
                    }
                }
            }

            for (let i = 0; i < allObjectsSorted.length; i++) {
                images[i] = document.createElement('img');
                images[i].setAttribute('src', allObjectsSorted[i].flag);
                images[i].setAttribute('width', '50');
                images[i].setAttribute('height', '30');
                names[i] = document.createElement('h4');
                names[i].textContent = allObjectsSorted[i].name;
                listNameAndFlag[i] = document.createElement('li');
                list1.appendChild(listNameAndFlag[i])
                listNameAndFlag[i].appendChild(images[i])
                listNameAndFlag[i].appendChild(names[i])
                listNameAndFlag[i].setAttribute('style', 'display: flex; align-items:center; margin-bottom: -25px;')
                population[i] = document.createElement('p');
                population[i].textContent = allObjectsSorted[i].name + ' has a total population of: ' + allObjectsSorted[i].population + ' people!';
                listPopulation[i] = document.createElement('li');
                list1.appendChild(listPopulation[i])
                listPopulation[i].appendChild(population[i]);


                region[i] = allObjectsSorted[i].region
                subregion[i] = allObjectsSorted[i].subregion
                colorCountriesPerContinent(region, i, subregion)
            }
        }
        makeCountryList();

    } catch (e) {
        console.error(e);
    }
}

getAllCountries()